pkg/gitrepo/repo_test.go

```diff
@@ -317,3 +324,90 @@ func Test_Repo_ResolveVersion(t *testing.T) {
 		})
 	}
 }
+
+// Test_Repo_GetFileContent tests Repo.GetFileContent method which retrieves
+// the content of a file.
+//
+// Tested repository can be found here:
+//
+//	https://github.com/giantswarm/gitrepo-test.
```

Comment:
Yeah why not, even though I don't really understand the benefit of this.

Reply:
Also test won't pass until this PR is merged because `pkg/testdata/DCO` will be added to master via this PR.

Reply:
The benefit would be testing content of the same file. If the file in the testdata in this repo and the file in the other repo change this would be rather confusing.

Reply:
> Also test won't pass until this PR is merged because pkg/testdata/DCO will be added to master via this PR.

That kind of sucks. Maybe we can add another dummy file for testing. Content of this file could be instructions where the original is supposed to be. Like:

```
An **identical** copy of this file are supposed to be in:

- https://github.com/giantswarm/gitrepo pkg/gitrepo/testdata/test-file.md file
- https://github.com/giantswarm/gitrepo-test test-file.md file

It is used to test `gitrepo.Repo.GetFileContent` method.
```

Reply:
I would prefer to keep the current test as it is, reading the `DCO` file from `gitrepo-test` and comparing this with the golden file at `pkg/gitrepo/testdata/DCO`. That's how we have setup tests with golden files so far. When the file change in the `gitrepo-test` we can use the -update flag.

Reply:
Yeah what you're saying makes sense. I lost the meaning of the golden file test.

Can you please add the command to update the test to the godoc as we usually do?

Reply:
comment added